owncloud/phoenix
# Folder upload

## Description

Added folder upload

## Related Issue




- Fixes #485 

## How Has This Been Tested?



- press on new
- click on upload folder, select one
- check if all the folder (except empty folder got created) and the files got uploaded
- delete the folder
- reupload but this time use drag & drop
- alle folders and files should be created (also empty folders) correctly

## Screenshots (if appropriate):

## Types of changes


## Checklist:



## Open tasks:

- [ ] write acceptance tests
- [x] fix PROPFIND Request
